test/test.cpp

```diff
@@ -31,7 +31,7 @@ int main()
     check_uuid_linkage();
     {
         IDXCoreAdapterFactory *factory = nullptr;
-        if (FAILED(DXCoreCreateAdapterFactory(&factory)))
+        if (FAILED(DXCoreCreateAdapterFactory(IID_PPV_ARGS(&factory))))

```

Comment:
`clang failed with stderr: C:\code\cpp\DirectX-Headers\test\test.cpp:34:20: error: no matching function for call to 'DXCoreCreateAdapterFactory'`

Reply:
This should be calling into:
https://github.com/microsoft/DirectX-Headers/blob/main/include/directx/dxcore.h#L27